tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -46,11 +46,6 @@
 SKIP_SAR_TESTS = RUNNING_ON_CI and RUNNING_TEST_FOR_MASTER_ON_CI and not RUN_BY_CANARY
 
 
-@skipIf(
-    # Hits public ECR pull limitation, move it to canary tests
-    ((not RUN_BY_CANARY) or (IS_WINDOWS and RUNNING_ON_CI) and not CI_OVERRIDE),
-    "Skip build tests on windows when running in CI unless overridden",
-)
```

Comment:
we don't want to skip anymore?

Reply:
no, it is my bad .. I removed them to run testing in my local machine, and forget to return them back :(